ocp_resources/sealed_secret.py

```diff
@@ -50,4 +50,4 @@ def to_dict(self) -> None:
 
     @property
     def keys_to_hash(self):
-        return ["spec..data", "spec..encryptedData"]
+        return ["data", "encryptedData"]
```

Comment:
This change the logic of the original usage
Now every `data` key will be hashed, not only the one under `spec`
we need to think if this is make sense or not.

Reply:
i think that we should be implicit here and keep the current logic

Reply:
Do we have any instance of same key? As far as I have seen, sometimes a key in spec might be present in status as well (in this case both should be hidden) or in case of templates and resource using templates, duplicate keys might be present as well (this might be a case where I don't know if hiding all is smart or not). But I need both your guidance on this one a bit more. I am a bit blind about my own usecase and have a mindset of `show only what we must` if not specified.

Reply:
it doesn't matter if we have the same key or not, maybe we do not have it now but what about later?

For example this is bad

```
In [11]: qq = {"a": ["1", "2"], "b": {"b1": [1,2]}}

In [12]: change_dict_value_to_hashed(qq, "b")
Out[12]: {'a': ['1', '2'], 'b': '******'}

In [13]: change_dict_value_to_hashed(qq, "a")
Out[13]: {'a': '******', 'b': '******'}

In [14]: change_dict_value_to_hashed(qq, "2")
Out[14]: {'a': '******', 'b': '******'}
```

We should be able to send `a.1` to mask only `1` under `a`

WDYT?

Reply:
ok, so may be if a.1 is passed then only mask a.1 else if "a" is passed, then mask all occurrences of "a"?

Reply:
Ok, I will update the PR based on our discussion today. Thanks a lot!